### PR TITLE
Add documentation for Srp Energy

### PR DESCRIPTION
**Pull request by briglx**
**Description:**
Add Documentation for Srp Energy.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#18036

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.srpenergy.markdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: "SRP Engery Sensor"
```
Comment (MartinHjelmare) [inline source/_components/sensor.srpenergy.markdown]: Energy

File: source/_components/sensor.srpenergy.markdown [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+description: "How to integrate SRP Energy within Home Assistant."
+date: 2018-10-30 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Energy
+featured: false
```
Comment (MartinHjelmare) [inline source/_components/sensor.srpenergy.markdown]: This is the default.

File: source/_components/sensor.srpenergy.markdown [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ha_release: "0.81"
```
Comment (MartinHjelmare) [inline source/_components/sensor.srpenergy.markdown]: 0.83

File: source/_components/sensor.srpenergy.markdown [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: srp_energy
+    username: YOUR_USERNAME
+    password: YOUR_PASSWORD
+    id: YOUR_ACCOUNT_ID
+```
+
+Configuration variables:
```
Comment (MartinHjelmare) [inline source/_components/sensor.srpenergy.markdown]: Use our new configuration tags for the config spec.

https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

File: source/_components/sensor.srpenergy.markdown [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+layout: page
+title: "SRP Energy Sensor"
+description: "How to integrate SRP Energy within Home Assistant."
+date: 2018-10-30 08:00
+sidebar: true
+comments: false
+ha_category: Energy
+ha_release: "0.83"
+ha_iot_class: "Cloud Polling"
+---
```
Comment (MartinHjelmare) [inline source/_components/sensor.srpenergy.markdown:11]: Is there no logo for this sensor brand?

Comment (briglx) [reply source/_components/sensor.srpenergy.markdown:11]: no logo

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**